projects/EWTeam/src/ServerInterface.java

```diff
@@ -133,6 +130,7 @@ private ServerInterface() {
 			StaticLog.info("Found value for scoreboard2password in pc2v9.ini");
 			pc2Properties = null;
 		} catch (Exception e) {
+			StaticLog.unclassified("Error loading pc2v9.ini "+ e.getMessage(), e);
```

Comment:
It does not seem to me that "unclassified" is an appropriate log message type for this condition. Failure to load a pc2v9.ini with a scoreboard password seems like it should be a "warning".

Reply:
I agree and in effect unclassified is a warning - take a look at the unclassified method.
I assume unclassified was used to indicate that this log level has not been classified yet,
there are a number of other logs that are unclassified.   
You alao indicated that you thought it was a good idea to minimize the time spent on these changes.